### ISSUE TITLE
`inline-verify --verifications-out` creates a file but it has no content

### ISSUE THREAD

**Issue by dkg** [pgpainless/sop-java#15]
When `pgpainless-cli inline-verify` has a `--verifications-out` argument, it creates the file, but doesn't write the expected verification string in it:

```
pgpainless-cli generate-key test > test.key
pgpainless-cli extract-cert < test.key > test.cert
echo test> test.txt
rm -f verifications
pgpainless-cli inline-sign test.key < test.txt > test.txt.pgp
pgpainless-cli inline-verify --verifications-out verifications test.cert < test.txt.pgp
ls -l verifications
```

What i'm seeing is that `verifications` is a zero-byte file.  It should have a line of verification info in it.

I still don't have a good sense of which project to open bugs in, so feel free to port this issue if i've got it in the wrong place.

## version info

```
$ pgpainless-cli version --extended
PGPainless-SOP 1.3.7
https://codeberg.org/PGPainless/pgpainless/src/branch/master/pgpainless-sop

Implementation of the Stateless OpenPGP Protocol Version 04
https://datatracker.ietf.org/doc/html/draft-dkg-openpgp-stateless-cli-04

Based on pgpainless-core 1.3.7
https://pgpainless.org

Using Bouncy Castle 1.71
https://www.bouncycastle.org/java.html
```

**Comment by vanitasvitae** [comment]
Oh boy, `pgpainless-cli` is more broken than I thought :O
Apparently my idea of using the interoperability test suite as a test suite for pgpainless-cli was not sufficient :D

I will look into it and try to get a fix out asap!

**Comment by vanitasvitae** [comment]
Annoyingly this is simply just a writer that was not flushed :angry:. Luckily the fix is easy.

**Comment by vanitasvitae** [comment]
Fixed with 8494f413e63e01614c0a381ec9c6fb0821793f84
